inc/packages/namespace.php

```diff
@@ -724,4 +725,106 @@ function maybe_add_accept_header( $args, $url ) : array {
 	return $args;
 }
 
+/**
+ * Validate the package alias for a DID.
+ *
+ * Uses `fair://` aliases from the DID document to determine the alias for the
+ * package. Performs bidirectional validation using DNS to ensure the DID is
+ * valid for the given alias.
+ *
+ * Uses cached result for one hour.
+ *
+ * @return string|WP_Error|null Alias domain if successfully validated, null if no valid alias is set, or error otherwise.
+ */
+function validate_package_alias( DIDDocument $did ) {
+	$cache_key = sprintf( 'fair_did_alias_' . $did->id );
```

Comment:
Why use `sprintf`?

Reply:
whoops, should have been `sprintf( 'fair_did_alias_%s', $did->id )`